pyi.py

```diff
@@ -246,6 +246,31 @@ def _check_platform_check(self, node: ast.Compare) -> None:
         else:
             self.error(node, Y007)
 
+    def visit_FunctionDef(self, node: ast.FunctionDef) -> None:
+        self.generic_visit(node)
+
+        for i, statement in enumerate(node.body):
+            if i == 0:
+                # normally, should just be "..."
+                if isinstance(statement, ast.Pass):
+                    # should disable this if https://github.com/python/typeshed/pull/1009 isn't
+                    # accepted
```

Comment:
Well, it *was* merged so you can remove this comment.

In general, it is perfectly fine for us to introduce checks here that *don't* apply to typeshed. Thanks to Flake8, we can disable those warnings in our config. For example, Mercurial is [starting to use stubs to help with migration to Python 3](https://www.mercurial-scm.org/wiki/TypeChecking). Function variables sometimes need annotation, too, so their config will disable Y010. This is fine.